### ISSUE TITLE
Distributed Model Parallel

### ISSUE THREAD

**Issue by adam-peaston-SC** [StrongResearch/isc-demos#40]
### Source / repo
https://pytorch.org/tutorials/intermediate/model_parallel_tutorial.html

### Model description
[DESCRIPTION]

### Dataset
[DATASET]

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]
